app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    author = User.find(params['like']['author_id'].to_i)
```

Comment:
- [ ] A like is created on behalf of the currently logged in user (current_user) of the app. Kindly make use of the current user of the app to create a like to posts.